charts/hoprd-operator/templates/crd-cluster-hoprd.yaml

```diff
@@ -25,6 +25,30 @@ spec:
     - name: v1alpha
```

Comment:
```suggestion
    - name: v1alpha2
```

Reply:
Pls iterate the API versions on these changes as well, or it will be a mess. Is it alpha? Or beta?

Reply:
Changed in 659b78d86df87f5a5d5408cde81ace9b67612d88